### ISSUE TITLE
Bugs in Cnchi, minimal iso, Plasma

### ISSUE THREAD

**Issue by angry-snake** [Antergos/antergos-iso#117]
Minimal ISO: Problems connecting network. It connects/disconnects, allows to type password not from the first time (takes time to detect Wi-Fi to work correctly, doesn't show Security page sometimes. CNCHI: If native language selected, you will type in native language with no posibility to switch, so you can't type name/username in English. When system has been installed you will have only one keyboard layout. Please make that if native language selected, system will be installed with native and English keyboard layout. Antergos KDE: Doesn't save Wi-Fi password. Connects/disconnects constantly until you connect manually or it just connects/disconnects twice until you type in your password. Package installer: If you was searching something and then cleared search bar it still says "Package not found". Menu - Alternatives: Please change to Classic menu by default.

**Comment by karasu** [comment]
The graphical package manager is called pamac, and it's not created by us. See here:
https://gitlab.manjaro.org/applications/pamac

>Menu - Alternatives: Please change to Classic menu by default.

Why? Does not work? If this is this a matter of taste, there will be users that prefer this one and others that won't. It's impossible to please everyone ;)

**Comment by angry-snake** [comment]
No, it's working but classic menu is simple and more convenient.
